.circleci/config.yml

```diff
@@ -1,976 +1,11 @@
-          context: org-global
-          requires:
-            - setup
-          filters:
-            tags:
-              only: /.*/
-            branches:
-              ignore:
-                - /feature*/
-                - /bugfix*/
-      - audit-licenses:
-          context: org-global
-          requires:
-            - setup
-          filters:
-            tags:
-              only: /.*/
-            branches:
-              ignore:
-                - /feature*/
-                - /bugfix*/
-      - license-scan:
-          context: org-global
-          requires:
-            - build-local
-          filters:
-            tags:
-              only: /v[0-9]+(\.[0-9]+)*(\-snapshot(\.[0-9]+)?)?(\-hotfix(\.[0-9]+)?)?(\-perf(\.[0-9]+)?)?/
-            branches:
-              ignore:
-                - /.*/
-      - image-scan:
-          context: org-global
-          requires:
-            - build-local
-          filters:
-            tags:
-              only: /v[0-9]+(\.[0-9]+)*(\-snapshot(\.[0-9]+)?)?(\-hotfix(\.[0-9]+)?)?(\-perf(\.[0-9]+)?)?/
-            branches:
-              ignore:
-                - /.*/
-      # New commits to main release automatically
-      - release:
-          context: org-global
-          requires:
-            - build-local
-            - pr-tools/pr-title-check
-            - test-lint
-            - test-unit
-            - test-coverage
-            - test-integration
-            - test-functional
-            - vulnerability-check
-            - audit-licenses
-            - license-scan
-            - image-scan
-          filters:
-            branches:
-              only:
-                - main
-                - /release\/v.*/
-      - github-release:
-          context: org-global
-          requires:
-            - release
-          filters:
-            branches:
-              only:
-                - main
-                - /release\/v.*/
-      - publish:
-          context: org-global
-          requires:
-            - build-local
-            - pr-tools/pr-title-check
-            - test-lint
-            - test-unit
-            - test-coverage
-            - test-integration
-            - test-functional
-            - vulnerability-check
-            - audit-licenses
-            - test-integration
-            - license-scan
-            - image-scan
-          filters:
-            tags:
-              only: /v[0-9]+(\.[0-9]+)*/
-            branches:
-              ignore:
-                - /.*/
-      - publish-snapshot:
-          context: org-global
-          requires:
-            - build-local
-            - pr-tools/pr-title-check
-            - test-lint
-            - test-unit
-            - test-coverage
-            - test-integration
-            - test-functional
-            - vulnerability-check
-            - audit-licenses
-            - test-integration
-            - license-scan
-            - image-scan
+      - build/workflow:
```

Comment:
Does this cover all the existing steps? example license scans, image scans? It needs to at least cover every existing CI step..

Please confirm

Reply:
Yes, but you can also check in CircleCI to confirm.
For example https://app.circleci.com/pipelines/github/mojaloop/ml-api-adapter/1158/workflows/96e01bfd-61c8-4fff-b360-c329cb5bbfee

Reply:
Thanks @kalinkrustev